.github/workflows/release.yml

```
@@ -0,0 +1,37 @@
+name: Create a new release when a release PR is merged
+
+# This action will only run when a pull request targeting `trunk` is closed/merged.
+on:
+  pull_request_target:
+    branches:
+      - 'trunk'
+    types:
+      - closed
+
+jobs:
+  create-release:
+    # Action will be skipped unless we have a successful merge AND the original PR started with release
+    # If this condition isn't matched, then github will show the action as skipped. 
+    if:  startsWith(github.head_ref, 'release') && github.event.pull_request.merged == true
```

```suggestion
    if:  startsWith(github.head_ref, 'release/') && github.event.pull_request.merged == true
```
thanks @shendy-a8c at the time I wasn't sure if we were using `release/` or `releases/` but agree with you. Better to be stricter.